src/GitHubCreateMergePRs/config.xml

```diff
@@ -16,13 +16,11 @@
     <!-- Roslyn release branches (flowing between releases) -->
     <merge from="release/dev17.10" to="main" />
 
-    <!-- Roslyn feature branches -->
-    <merge from="main" to="features/semi-auto-props" owners="333fred" frequency="weekly" />
-    <merge from="main" to="features/vs2022-theme" owners="joerobich" frequency="weekly" />
-    <merge from="main" to="features/editorconfig_lsp" owners="dibarbet,Danyboyyy" frequency="weekly" />
-    <merge from="main" to="features/FixAllNamingViolation" owners="Cosifne" frequency="weekly" />
```

Comment:
@Cosifne to confirm removal of this and `features/lsp_configuration` for auto merge configuration

Reply:
Yes, that branch is no longer needed